mixin-styles-gb/js/block-variations.js

```diff
@@ -0,0 +1,27 @@
+// Register new block variations
```

Comment:
This is not needed anymore. Define the template area and template parts in experimental-theme.json https://github.com/WordPress/gutenberg/pull/30118/